File: ProgramingBasics/AdvancedLoops/09.SumDigits/src/SumDigits.java
import java.util.Scanner;

public class SumDigits {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = Integer.parseInt(scanner.nextLine());
        int sum = 0;

        do {
            sum += (n % 10);
            n /= 10;
        } while (n > 0);

        System.out.println(sum);
    }
}

File: ProgramingBasics/AdvancedLoops/09.SumDigits/09.SumDigits/SumDigits.cs
using System;

namespace _09.SumDigits
{
    public class SumDigits
    {
        public static void Main(string[] args)
        {
            var n = int.Parse(Console.ReadLine());
            var sum = 0;

            do
            {
                sum += (n % 10);
                n /= 10;
            } while (n > 0);

            Console.WriteLine(sum);
        }
    }
}
